### ISSUE TITLE
Localization not applied

### ISSUE THREAD

**Issue by r3try3** [Shmew/Feliz.MaterialUI.MaterialTable#5]
## Description
Some localization options for the toolbar are not applied correctly (always shown in english).

## Steps to reproduce
1. Go to the [example page for localization ](https://shmew.github.io/Feliz.MaterialUI.MaterialTable/#/MaterialTable/Examples/Localization)
2. Navigate to Examples > Localization
2. hover over the search icon
3. see that the given translation for the searchTooltip is not applied

## Expected behavior
Tooltip for Search-Icon should be "Lêgerîn" but is actually "Search"

## Screenshots
![image](https://user-images.githubusercontent.com/72866041/107483576-90d89b80-6b81-11eb-8dd6-3721c7588f8f.png)

## Additional context
Code from example page:
```
materialTable.localization [
   localization.toolbar [
     localization.toolbar.searchTooltip "Lêgerîn"
   ]
]
```

**Comment by r3try3** [comment]
seems to also not work for the following properties:
```
localization.toolbar.searchPlaceholder
localization.toolbar.addRemoveColumns
localization.toolbar.showColumnsTitle
localization.toolbar.showColumnsAriaLabel
```
for now i have only found issues with properties within localization.toolbar (e.g. localization.pagination.* works as intended)

**Comment by Shmew** [comment]
Thanks for the bug report! This is now fixed with 1.8.4, which will be indexed on NuGet shortly.
